Description: Add missing headers to fix undefined reference to `S_ISREG' linker
 error and hence FTBFS
Author: Bhavani Shankar <bhavi@ubuntu.com>
Bug-Ubuntu: https://bugs.launchpad.net/bugs/600129
Index: audit-1.7.13/lib/libaudit.c
===========================================
--- audit-1.7.13.orig/lib/libaudit.c
+++ audit-1.7.13/lib/libaudit.c
@@ -36,6 +36,8 @@
 #include <sys/utsname.h>
 #include <fcntl.h>	/* O_NOFOLLOW needs gnu defined */
 #include <limits.h>	/* for PATH_MAX */
+#include <sys/stat.h>
+#include <sys/types.h>
 
 #include "libaudit.h"
 #include "private.h"
